hw/ip/kmac/doc/_index.md

```diff
@@ -258,12 +258,11 @@ However, EDN provides the entropy in much less size.
 Requesting multiple entropy to EDN module degrades the KMAC throughput.
 This module requests the entropy to EDN then expands the entropy to 1600 bit.
 
-This module, at first, expands the generated random values to 320 bits by stacking the values in every cycle.
-Then the 320b of the entropy is duplicated into 1600bit.
-Internally, the module stores only 320bit of entropy.
-We choose 320bit to cover one plane (5x 64bit).
+This module, at first, expands the generated random values to 800 bits by stacking the values in every cycle.
+Then the 800b of the entropy is duplicated into 1600bit.
```

Comment:
wait is this duplication still true? I thought because DOM is half size now, the 800b is exactly what we need, did I misunderstand?

Reply:
Ah sorry, I should have removed this. I still need to update the documentation and will do this once there is agreement we want to move forward with this PR.

Reply:
I've now also aligned the documentation.